CodeConverter/CSharp/CommonConversions.cs

```diff
@@ -701,7 +766,19 @@ public static ExpressionSyntax NothingComparison(ExpressionSyntax otherArgument,
                     SyntaxFactory.ConstantPattern(ValidSyntaxFactory.NullExpression));
             }
 
-            return SyntaxFactory.BinaryExpression(CSSyntaxKind.EqualsExpression, otherArgument, ValidSyntaxFactory.DefaultExpression);
+            return SyntaxFactory.BinaryExpression(CSSyntaxKind.EqualsExpression, otherArgument,
+                ValidSyntaxFactory.DefaultExpression);
+        }
+
+        public async Task<string> GetClassificationLastTokenAsync(VBSyntax.SimpleImportsClauseSyntax clause)
+        {
+            // Global imports aren't associated with a document and no need to classify aliases
+            if (clause.SyntaxTree.FilePath == string.Empty || clause.Alias != null) { return null;}
+
+            var span = clause.GetLastToken().Span;
+            var spans = await Classifier.GetClassifiedSpansAsync(Document, span);
+
+            return spans.Last().ClassificationType;
```

Comment:
This is cool, I didn't know about it and probably could have used it several other places, nice!